packages/eslint-config/index.js

```diff
@@ -1,14 +1,13 @@
 module.exports = {
 	parser: 'babel-eslint',
 	extends: [
-		'wordpress',
-		'plugin:wordpress/esnext',
+		'./configs/wordpress.js',
+		'./configs/esnext.js',
```

Comment:
Do we really need separate files here? Or maybe we should rename `wordpress.js` into `es5.js` if that's the distinction between the two?